onnxruntime/core/providers/webnn/builders/model.cc

```diff
@@ -36,20 +36,18 @@ Status Model::Predict(const InlinedHashMap<std::string, OnnxTensorData>& inputs,
     }
     auto num_elements = SafeInt<size_t>(Product(tensor.tensor_info.shape));
     emscripten::val view{emscripten::typed_memory_view(num_elements, static_cast<const float*>(tensor.buffer))};
-    // WebNN API only accepts non-shared ArrayBufferView.
-    // https://webmachinelearning.github.io/webnn/#typedefdef-mlnamedarraybufferviews
-    emscripten::val SharedArrayBuffer = emscripten::val::global("SharedArrayBuffer");
-    if (SharedArrayBuffer.as<bool>()) {
-      emscripten::val non_shared_array = emscripten::val::global("Float32Array").new_(static_cast<uint32_t>(num_elements));
-      non_shared_array.call<void>("set", view);
-      wnn_inputs_.set(name, non_shared_array);
-    } else {
-      wnn_inputs_.set(name, view);
-    }
+    // Workaround for WebAssembly multi-threads enabled since WebNN API only accepts non-shared ArrayBufferView.
+    // https://www.w3.org/TR/webnn/#typedefdef-mlnamedarraybufferviews
```

Comment:
nit: move the above two lines of comments into the `#ifdef ENABLE_WEBASSEMBLY_THREADS` section.